ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -160,10 +159,12 @@ public static BeaconState get_initial_beacon_state(
       index++;
     }
 
+    List<Validator> activeValidators =
+        ValidatorsUtil.get_active_validators(
+            state.getValidator_registry(), UnsignedLong.valueOf(GENESIS_EPOCH));
     Bytes32 genesis_active_index_root =
-        validatorListHashTreeRoot(
-            ValidatorsUtil.get_active_validators(
-                state.getValidator_registry(), UnsignedLong.valueOf(GENESIS_EPOCH)));
+        hash_tree_root(
+            activeValidators.stream().map(item -> item.toBytes()).collect(Collectors.toList()));
```

Comment:
I had changed this in my most recent PR. This is not in line with spec because get_active_validators return List of validators, where spec tells us to hash_tree_root list of integers

Reply:
Ahhh ok cool. I would have figured that out eventually once I reviewed your PR

Reply:
Yeah, hash_tree_root on a List<Integer> is a little bit harder. I've since merged in a method in TreeHashUtil/HashTreeUtil to do it for us, and plan on making it a little bit easier with a fix in Cava. See https://github.com/ConsenSys/cava/issues/184.

Reply:
rebased from master and now using the new `integerListHashTreeRoot`